snowfakery/data_generator_runtime.py

```diff
@@ -569,7 +569,11 @@ def executable_blocks(self):
         return {**self.field_funcs(), "fake": self.fake}
 
     def fake(self, name):
-        return str(self.runtime_context.faker_template_library._get_fake_data(name))
+        val = self.runtime_context.faker_template_library._get_fake_data(name)
```

Comment:
Previously the faker function serialized everything to strings. Now it does not. The user can still do so if they want to, by using jinja filters.